rasa_sdk/endpoint.py

```diff
@@ -90,7 +92,13 @@ async def health(_) -> HTTPResponse:
     @app.post("/webhook")
     async def webhook(request: Request) -> HTTPResponse:
```

Comment:
[the test is missing](https://github.com/RasaHQ/rasa-sdk/pull/925/files#diff-adf6af4744d4ee23f8b03b6fddcd7ca0ae330cd2ba8e3dc39c834ee76dbe47de)